### ISSUE TITLE
Add python3-M2Crypto package in Opensuse 15.1 bootstrap repository

### ISSUE THREAD

**Issue by phibid** [uyuni-project/uyuni#2124]
Uyuni release 2020.03

Bootstrap an Opensuse 15.1 Salt client is failing due to `python3-M2Crypto` missing package:

```
Problem: nothing provides python3-M2Crypto needed by python3-salt-2019.2.0-lp151.5.12.1.x86_64
 Solution 1: do not install salt-2019.2.0-lp151.5.12.1.x86_64
 Solution 2: break python3-salt-2019.2.0-lp151.5.12.1.x86_64 by ignoring some of its dependencies
```

Checking the configuration of the bootstrap repository for opensuse 15.1 in `/usr/share/susemanager/mgr_bootstrap_data.py`, I notice this:

```
PKGLIST15_SALT = [
    "libpgm-5_2-0",
    "libsodium23",
    "libzmq5",
    "python3-Babel",
    "python3-certifi",
    "python3-chardet",
    "python3-idna",
    "python3-Jinja2",
    "python3-MarkupSafe",
    "python3-msgpack",
    "python3-psutil",
    "python3-py",
    "python3-pycrypto|python3-M2Crypto",
```

And on my bootstrap repo, only python3-pycrypto is present. Not sure why there is a pipe between `python3-pycrypto` and `python3-M2Crypto`, but if we want to have this bootstrap repo working for both traditional and Salt clients, both libraries should be presents.

Meanwhile, directly specifying `python3-M2Crypto` as argument of `mgr-create-bootstrap-repo` solve my issue.

Thanks !

**Comment by juliogonzalez** [comment]
It seems this is a leftover from a salt migration, as some versions required one and some others the other.

As of now all SLE15 versions and Leap 15.1 versions require python3-M2Crypto, so I will adjust it.

This will break Leap 15.0 which still requires python3-pycrypto, but it's already EoL for more than 4 months now.

**Comment by juliogonzalez** [comment]
This fix will be part of 2020.04 which I hope to release today.
